languages/civicrm.pot

```diff
@@ -1,192 +1,115 @@
-# Copyright (C) 2014 CiviCRM
+# Copyright (C) 2020 CiviCRM
 # This file is distributed under the same license as the CiviCRM package.
 msgid ""
 msgstr ""
-"Project-Id-Version: CiviCRM 4.6\n"
-"Report-Msgid-Bugs-To: http://wordpress.org/tag/civicrm\n"
-"POT-Creation-Date: 2014-11-11 09:48:56+00:00\n"
+"Project-Id-Version: CiviCRM 4.7\n"
+"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/civicrm\n"
```

Comment:
@christianwach this link is not active.  Is this in anticipation of the plugin repo accepting the submission?

Reply:
@kcristiano Please see [my reply to Joe](https://github.com/civicrm/civicrm-wordpress/pull/182#discussion_r379571956) which answers this.

Reply:
Got it.